src/util/models.py

```diff
@@ -464,15 +464,15 @@ def merge_model(cr, source, target, drop_table=True, fields_mapping=None, ignore
         if ir.res_model and not ir.res_id and ir.table not in ignores:
             # only update unbound references, other ones have been updated by the call to
             # `replace_record_references_batch`
-            wheres = []
+            where_clauses = []
```

Comment:
I'm not fan of this code changes. Are there automatic? Can we restrict the fixes to comments (or docstrings?) and perhaps general strings?

Reply:
Not automatic, but `typos` didn't like the word `wheres`

And I don't think we can restrict to only comments.

Reply:
This one is kind of OK. But I'm still no OK with `fpt` or `TrUe`. It really feels like people will complain about this CI being too intrusive... with reason :/